language/move-prover/spec-lang/src/translate.rs

```diff
@@ -630,6 +630,7 @@ impl<'env, 'translator> ModuleTranslator<'env, 'translator> {
     /// Creates a SpecBlockContext from the given SpecBlockTarget
     fn get_spec_block_context(&self, target: &PA::SpecBlockTarget) -> Option<SpecBlockContext> {
         match &target.value {
+            PA::SpecBlockTarget_::Code => unreachable!(),
```

Comment:
How is it unreachable?

Reply:
They are not. Boogie translation of these expressions will be done in a separate PR.